### PR TITLE
Consistent HTML formatting

### PR DESCRIPTION
**Pull request by decentralgabe**
No normative changes. Purely editorial for consistency.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/315.html" title="Last updated on Oct 10, 2024, 6:11 PM UTC (5e818a1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/315/cc317ae...5e818a1.html" title="Last updated on Oct 10, 2024, 6:11 PM UTC (5e818a1)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,459 +1,451 @@
...
         conforming to the Verifiable Credential data model [[VC-DATA-MODEL-2.0]]
-        with JSON Object Signing and Encryption (<a href="https://datatracker.ietf.org/wg/jose/about/">JOSE</a>),
-        Selective Disclosure for JWTs [[SD-JWT]],
-        and CBOR Object Signing and Encryption (COSE) [[RFC9052]].
-        This enables the Verifiable Credential data model
-        [[VC-DATA-MODEL-2.0]] to be implemented with standards
+        with JSON Object Signing and Encryption 
+        (<a href="https://datatracker.ietf.org/wg/jose/about/">JOSE</a>),
+        Selective Disclosure for JWTs [[SD-JWT]], and CBOR Object Signing and 
+        Encryption (COSE) [[RFC9052]]. This enables the Verifiable Credential
```
Comment (selfissued) [inline index.html:131]: "CBOR Object Signing and Encryption" is broken across lines, when it's a term that should be searchable and so needs to be on a single line.

File: index.html [hunk 2]
```diff
@@ -1,459 +1,451 @@
...
-                    status: "Special Publication 800-122",
-                    publisher: "NIST"
-                },
-            }
-        };
+      // See https://github.com/w3c/respec/wiki/ for how to configure
+      // ReSpec
+      var respecConfig = {
+          group: "vc",
+
```
Comment (selfissued) [inline index.html:13]: It's better not to have any whitespace at the end of lines.

**Review by selfissued — CHANGES_REQUESTED** [review]
If we're going to change the line breaks in the source, we should be employing [Semantic Line Breaks](https://sembr.org/), so that the source can easily be searched for terms because the terms are never broken across lines.

The simple form of these rules is that

1. Line breaks should never occur within a sentence, except after punctuation marks, such as commas.
2. There should always be a line break after periods ending sentences (but not those indicating abbreviations).

Before we merge this, I'd also like to understand how we are going to verify that no changes to the output occurred.  (When doing changes like this, it's easy for stray characters to creep in or be deleted due to manual errors.)

Thanks.

**Review by selfissued — APPROVED** [review]
Looks good now

**Comment by decentralgabe** [comment]
only formatting changes. merging.

**Pull request merged**